packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -327,7 +327,7 @@ let _redirectHandler: ((event: RedirectEvent) => void) | null = null;
 
 // Store the `resolve` function from a Promise to fire when the AppState
 // returns to active
-let _onWebBrowserCloseAndroid: null | (() => void) = null;
+let _onWebBrowserCloseAndroid: null | ((value?: any) => void) = null;
```

Comment:
resolve typing was updated: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-1.html#resolves-parameters-are-no-longer-optional-in-promises